cmasher/utils.py

```diff
@@ -233,6 +239,102 @@ def _get_cmap_perceptual_rank(
 
 
 # %% FUNCTIONS
+# This function combines multiple colormaps at given nodes
+def combine_cmaps(
+    cmaps: list[Union[Colormap, str]],
+    nodes: Optional[list[float]] = None,
+    n_rgb_levels: int = 256,
+) -> LinearSegmentedColormap:
+    """Create a composite matplotlib colormap by combining multiple colormaps.
+
+    Parameters:
+        cmaps (list[Union[Colormap, str]]): list of colormaps to combine.
```

Comment:
type-annotation syntax should be avoided in docstrings because it doesn't age well and we don't have tooling to auto-upgrade it to newer syntax when they become available. We also usually reserve the inline comment to typing information, and the description gets its own line
```suggestion
        cmaps: list of Colormap and colormap names (str)
            List of colormaps to combine.
```
I've added a suggestion for this argument only by you'd need to do the same thing for all three.

Reply:
At least I personally don't know of a good tool to check type annotations in docstring either, and I agree it could be hard to keep the type in code and docstring synced. 

So the ladder of goodness would be :-) :
```
Heaven

Good and updated type annotations in docstring.

No type annotations.

Wrong or outdated type annotations.
```
I would fall back to no type annotations for now until I find a good tool to check type annotations in docstring.

Reply:
Please keep your type annotations ! I'm just suggesting to avoid using Python syntax for type information in doc strings. As you saw, we do have tooling to check actual annotations

Reply:
Okay sure I understand you now...